cognite/client/_api/data_modeling/graphql.py

```diff
@@ -133,3 +134,32 @@ def apply_dml(
         query_name = "upsertGraphQlDmlVersion"
         res = self._post_graphql(url_path="/dml/graphql", query_name=query_name, json=payload)
         return DMLApplyResult.load(res[query_name]["result"])
+
+    def query(self, id: DataModelIdentifier, query: str, variables: dict[str, Any] | None = None) -> dict[str, Any]:
+        """Apply the DML for a given data model.
```

Comment:
A bit cryptic for me at least what this means "Apply the DML for a given data model."

Reply:
Copy-paste blunder from the above method 😵‍💫